app/controllers/api/v1x2/order_processes_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1x2
+    class OrderProcessesController < ApplicationController
+      include Api::V1x1::Mixins::IndexMixin
+      include Api::V1x0::Mixins::ShowMixin
```

Comment:
Same thing here, I think we can probably just use `Mixins::IndexMixin` and `Mixins::ShowMixin`. I will fix this in a follow-up PR.